src/frontend/apps/drive/src/features/ui/preview/video-player/VideoPlayer.scss

```diff
@@ -28,15 +27,22 @@
   }
 
   &__video {
+    flex: 1; // Take available space
     width: 100%;
-    height: auto;
-    aspect-ratio: 16 / 9;
-    object-fit: cover;
+    height: 100%;
+    max-height: calc(100vh - 200px); // Leave space for controls
+    object-fit: contain; // Keep proportions
     display: block;
     border-radius: 4px;
 
     &__wrapper {
       position: relative;
+      max-height: calc(100vh - 200px); // Fixed height to avoid "jump"
```

Comment:
Is it really needed to repeat the max-height property as those two elements are nested?